stock_outgoing_shipment_report/models/stock_outgoing_shipment_report.py

```diff
@@ -14,7 +14,7 @@
     "product_name": ["Char", 32],
     "case_qty": ["Float", 5],
     "separate_qty": ["Float", 5],
-    "lot_num": ["Char", 10],
+    #"lot_num": ["Char", 10],
```

Comment:
Remove this line.  No need to leave it as a comment.

Reply:
ok I'll try it